docs/docusaurus/src/css/typography.scss

```diff
@@ -0,0 +1,85 @@
+:root {
+    --ifm-font-family-base: 'Source Sans Pro', 'Open Sans', Montserrat, Verdana, sans‑serif;
+
+    /* Variables based on style guide. */
+    --h1-font-size: 3rem;
+    --h2-font-size: 2rem;
+    --h3-font-size: 1.75rem;
+    --h4-font-size: 1.5rem;
+    --h5-font-size: 1.25rem;
+
+    --p-font-size: 1.125rem;
+    --p-lg-font-size: 1.5rem;
+    --p-md-font-size: 1rem;
+    --p-sm-font-size: 0.875rem;
+    --p-tiny-font-size: 0.75rem;
+
+    --p-line-height: 1.5;
+
+    /* Var Overwrites */
+    --ifm-h1-font-size: var(--h1-font-size);
+    --ifm-h2-font-size: var(--h2-font-size);
+    --ifm-h3-font-size: var(--h3-font-size);
+
+    --ifm-heading-line-height: 1.25;
+
+    --ifm-font-weight-semibold: 600;
+
+    --ifm-heading-margin-bottom: 0.3em;
+    --ifm-heading-vertical-rhythm-bottom: 0;
+
+    --ifm-list-item-margin: 0.5em;
+
+    --ifm-dropdown-font-weight: bold;
+}
+
+/* override huge h1 */
+/* TODO: this rule should probably be deleted but we need to check if it is still necessary */
+.h1Heading_node_modules-\@docusaurus-theme-classic-lib-next-theme-Heading-styles-module {
+    font-size: var(--ifm-h1-font-size) !important;
+}
+
+.markdown {
+   h1 {
+     --ifm-h1-font-size: var(--h1-font-size);
+   }
+
+   h2 {
+     --ifm-h2-font-size: var(--h2-font-size);
+   }
+
+   h3 {
+     --ifm-h3-font-size: var(--h3-font-size);
+   }
+ }
+
+h4 {
+    font-size: var(--h4-font-size);
+}
+
+h5 {
+    font-size: var(--h5-font-size);
+}
+
+h4, h5 {
+    line-height: var(--p-line-height);
+}
+
+p {
+    font-size: var(--p-font-size);
+    line-height: var(--p-line-height);
+}
+
+.markdown h1:first-child {
+    --ifm-h1-font-size: var(--h1-font-size);
+    margin-bottom: var(--ifm-heading-margin-bottom);
+}
+
+.markdown li,
+.markdown li p {
+    font-size: var(--p-font-size);
+}
+
+.markdown li p:only-child {
+    margin: 0;
+}
```

Comment:
we should be able to move these into the same `.markdown` block above.

Reply:
Sorry, I missed those! Should be fixed now :smile: